### ISSUE TITLE
Please release the latest version post #1113 merge

### ISSUE THREAD

**Issue by LearnGrowAndShare** [powermock/powermock#1127]
Hey there and thank you for using PowerMock

Please read the following tips before filing an issue:

Is this something you can debug and fix?
-------------------------------------------------------
No, as this has to be published by project owner.

Is this question about PowerMock usage?
---------------------------------------
No.

None of the above, want to create a GitHub issue
------------------------------------------------------------------

The current version 2.0.9 does not contains the below commit, or #1113 changes 
https://github.com/powermock/powermock/commit/a0be5c4bbeb04f13a6aec2653c1f5663cece1316

It would be a great help, if you can publish the next version with the new changes on release branch.

Thanks,

**Comment by anabright** [comment]
Yes please, can we please get a new release with the latest changes?

**Comment by albertocavalcante** [comment]
It would be great to have a newer release

**Comment by goldaxe** [comment]
Need support for mockito v4
It would be great if a new version could be released

**Comment by rakshitcgupta** [comment]
So powermock2 does not support mockito4 + or vice versa, so does that mean we now use functionality from mockitov4 and remove powermock from our code permanently?

**Comment by LearnGrowAndShare** [comment]
We migrated to mockito-inline..

**Comment by praetore** [comment]
I really want to see a new release.
